fs/artifact_fetcher.go

```diff
@@ -165,7 +166,7 @@ func (f *artifactFetcher) Store(ctx context.Context, desc ocispec.Descriptor, re
 	return nil
 }
 
-func FetchSociArtifacts(ctx context.Context, imageRef, indexDigest string, store content.Storage) (*soci.SociIndex, error) {
+func FetchSociArtifacts(ctx context.Context, imageRef, indexDigest string, store content.Storage) (*soci.Index, error) {
```

Comment:
Are the changes in this file related to the OCI references type support?  If so, how?

Reply:
Not really. As far as the artifact fetcher is concerned, it doesn't really need to know whether it's ORAS or OCI manifest, since the `Blob` field is the same in both of them. The changes are QOL changes made to avoid unnecessary copies.